hilti/runtime/include/types/null.h

```diff
@@ -4,21 +4,23 @@
 
 #include <arpa/inet.h>
 
-#include <optional>
 #include <ostream>
 #include <string>
 
 #include <hilti/rt/extension-points.h>
 
 namespace hilti::rt {
 
+template<typename T>
+class Optional;
+
 /**
  * Represents HILTI's "null" type.
  */
 struct Null {
     template<typename T>
-    operator std::optional<T>() {
-        return std::nullopt;
+    operator hilti::rt::Optional<T>() {
+        return {};
     }
 };
```

Comment:
`Optional` can already construct from a `Null` and this is not needed anymore.

```suggestion
struct Null {};
```